### PR TITLE
add types definition

### PR DESCRIPTION
**Pull request by acrazing**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "version": "2.1.0",
   "description": "Blowfish encryption library for browsers and Node.js",
   "main": "dist/blowfish.js",
+  "types": "./egoroof-blowfish.d.ts",
```
Comment (egoroof) [inline package.json]: maybe just `blowfish.d.ts` ?

Comment (acrazing) [reply package.json]: It's the file name of the module's .d.ts, just like `main` for commonjs.

If change this field, we need to change the file name of `egoroof-blowfish.d.ts` to `blowfish.d.ts` also.

And it effects nothing, I don't think it is necessary.

Comment (egoroof) [reply package.json]: yeah, I mean to change filename, as it seems unnecessary long

but it's not blocker, anyway

**Comment by masterkain** [comment]
how to use this?

```
import * as Blowfish from 'egoroof-blowfish'
const bf = new Blowfish(base64key, Blowfish.MODE.CBC, Blowfish.PADDING.PKCS5)
```

```
This expression is not constructable.
  Type 'typeof import("/myproject/node_modules/egoroof-blowfish/egoroof-blowfish")' has no construct signatures.
```

**Pull request merged**